_custom_build/backend.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+from setuptools.build_meta import *  # noqa: F401, F403
```

Comment:
What's being imported here? Can we name them?

Reply:
This started with https://setuptools.pypa.io/en/latest/build_meta.html#dynamic-build-dependencies-and-other-build-meta-tweaks, which says
> You can override any of the functions specified in [PEP 517](https://peps.python.org/pep-0517/#build-backend-interface), not only the ones responsible for gathering requirements. It is important to import * so that the hooks that you choose not to reimplement would be inherited from the setuptools’ backend automatically. This will also cover hooks that might be added in the future like the ones that [PEP 660](https://peps.python.org/pep-0660/) declares.

Reply:
I requested that clarification to the setuptools docs this spring, having experimented with the in-tree backends for years, after my colleague at Ansible pointed out the problem — so I've gone ahead and documented it.
Ref: https://github.com/pypa/setuptools/pull/3893.